File: backend/src/model/PlayerRole.java
package model;

public class PlayerRole {
    public static final String WORD_MASTER= "WORD_MASTER";
    public static final String GUESS_MASTER= "GUESS_MASTER";
    public static final String INDETERMINE= "INDETERMINE";

    public static boolean isValid(String role) {
        return role.equals(WORD_MASTER) || role.equals(GUESS_MASTER);
    }
}

File: frontend/js/utils/playerRole.js
export class PlayerRole {
    static #_WORD_MASTER = "WORD_MASTER";
    static #_UNDEFINED = "UNDEFINED";
    static #_GUESS_MASTER = "GUESS_MASTER";

    static get WORD_MASTER() { return this.#_WORD_MASTER; }
    static get UNDEFINED() { return this.#_UNDEFINED; }
    static get GUESS_MASTER() { return this.#_GUESS_MASTER; }
}